src/_pytest/fixtures.py

```diff
@@ -33,6 +33,7 @@
 from _pytest.outcomes import TEST_OUTCOME
 
 if False:  # TYPE_CHECKING
+    from _pytest import nodes
```

Comment:
Type annotations reveal many "semantic" import cycles. I think we should try to improve the situation some time!

BTW, for good measure, would move this import below the `from typing import Type` one.